test/e2e/command/botkube.go

```diff
@@ -97,20 +100,22 @@ func compareFilters(expected, actual []string) bool {
 func (c *context) testNotifierCommand(t *testing.T) {
 	// Disable notifier with @BotKube notifier stop
 	t.Run("disable notifier", func(t *testing.T) {
-		// Send message to a channel
-		c.SlackServer.SendMessageToBot(c.Config.Communications.Slack.Channel, "notifier stop")
+		if c.TestEnv.Config.Communications.Slack.Enabled {
+			// Send message to a channel
+			c.SlackServer.SendMessageToBot(c.Config.Communications.Slack.Channel, "notifier stop")
 
-		// Get last seen slack message
-		time.Sleep(time.Second)
-		lastSeenMsg := c.GetLastSeenSlackMessage()
-
-		// Convert text message into Slack message structure
-		m := slack.Message{}
-		err := json.Unmarshal([]byte(*lastSeenMsg), &m)
-		assert.NoError(t, err, "message should decode properly")
-		assert.Equal(t, c.Config.Communications.Slack.Channel, m.Channel)
-		assert.Equal(t, fmt.Sprintf("```Sure! I won't send you notifications from cluster '%s' anymore.```", c.Config.Settings.ClusterName), m.Text)
-		assert.Equal(t, config.Notify, false)
+			// Get last seen slack message
+			time.Sleep(time.Second)
```

Comment:
@aananthraj it might be a good idea to move Sleep statements inside `GetLastSeenSlackMessage()` function, it will be easier to maintain

Reply:
I guess we could add `go vet ./test/...` to `verify-gover.sh`

Reply:
pushed those changes